snap/snapcraft.yaml

```diff
@@ -183,6 +183,18 @@ parts:
         craftctl default
 
         mkdir -p "${CRAFT_PART_INSTALL}/etc/ovn/"
+    override-stage: |
+        craftctl default
+        pkg_version=$(
+            dpkg-deb -f \
+            $CRAFT_PART_SRC/../stage_packages/ovn-host*.deb \
+            Version | sed -rne 's/([0-9.]+)[-+].*$$/\1/p')
+        git_version=$(
+            git -C $CRAFT_PROJECT_DIR describe \
+                --always \
+                --dirty \
+                --abbrev=10 | sed -s 's/^v//')
```

Comment:
Purely out of curiosity, what's the purpose of this `sed`?

Reply:
Ah, right, it would be useful if we were tagging this repository, to ensure that the `v` in `v1.2.3` does not get exposed in the final version string, but since we are not tagging I can remove it.

Thank you for highlighting it!